api/envoy/admin/v3/clusters.proto

```diff
@@ -84,6 +84,9 @@ message ClusterStatus {
 
   // Observability name of the cluster.
   string observability_name = 7;
+
+  // eds_service_name from EDS config of the cluster.
```

Comment:
```suggestion
  // The :ref:`EDS service name <envoy_v3_api_field_config.cluster.v3.Cluster.EdsClusterConfig.service_name>` if the cluster is an EDS cluster.
```
(please double check that the reference I wrote above links to the right location.)